### ISSUE TITLE
Configure Swagger to Load DTO's From the @agoracloud/common Package

### ISSUE THREAD

**Issue by saidghamra** [AgoraCloud/server#79]
When I removed all the DTO's from the server, I forgot to configure Swagger to scan all DTO's available through the `@agoracloud/common` package. Currently, when the user navigates to Swagger UI, they are not able to see the schema of request and response bodies (i.e DTO's). For example:

![image](https://user-images.githubusercontent.com/35788699/122141548-8f62fc80-ce1b-11eb-88d1-3f8382096f86.png)

**Comment by saidghamra** [comment]
When all the DTO's were in the server, NestJs's `@nestjs/swagger` nest-cli plugin was automatically scanning all DTO's and generating all Swagger OpenAPI annotations and documentation.

Since the DTO's were moved to the `@agoracloud/common` package from the server, the `@nestjs/swagger` nest-cli plugin could not scan the DTO's. Therefore, I had to manually type out all the Swagger OpenAPI annotations to get them to display in the servers SwaggerUI.
